examples/subscribing-example-app/public/RiderConnection.js

```diff
@@ -64,7 +76,22 @@ export class RiderConnection {
       marker.focus();
     }
 
-    this.rider.move(locationCoordinate, this.shouldSnap);
+    if (this.timeouts.length) {
```

Comment:
Coercion to boolean here is a bit icky (if you want to be specific, then `> 0` would be more readable)... though I think this check is probably superfluous if `this.timeouts` is always defined.

Reply:
df4e743